types/node/test/http.ts

```diff
@@ -133,10 +133,13 @@ import * as net from 'net';
     const method: string = req.method;
 }
 
+// HTTP Methods
 {
-    // Status codes
-    let codeMessage: string = http.STATUS_CODES['400']!;
-    codeMessage = http.STATUS_CODES[400]!;
```

Comment:
can we bring this back, if this is not rewritten?
say as:
```ts
http.STATUS_CODES['400']; // $ExpectType string
http.STATUS_CODES[400]; // $ExpectType number
```
thx!

Reply:
Yes, my bad!